packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -450,12 +455,23 @@ export default class Dropdown extends React.Component<DropdownProps, State> {
             : 0;
 
         let focusCounter = 0;
+        let itemRole;
```

Comment:
NIT: This could be immutable (`const`) if the switch were in a `getRole` method.